test/metabase/sample_data_test.clj

```diff
@@ -147,3 +147,26 @@
               (testing "after"
                 (is (= []
                        (rating)))))))))))
+
+(deftest ddl-sample-database-test
+  (testing "should be able to execute DDL statements on the Sample Database"
+    (mt/with-temp Database [db (sample-database-db true)]
+      (sync/sync-database! db)
+      (mt/with-db db
+        (let [conn-spec (sql-jdbc.conn/db->pooled-connection-spec (mt/db))
+              get-tables (fn [] (set (mapv :table_name (jdbc/query conn-spec "SHOW TABLES;"))))
+              get-schemas (fn [] (set (mapv :schema_name (jdbc/query conn-spec "SHOW SCHEMAS;"))))]
+          (testing "create schema"
+            (is (not (contains? (get-schemas) "NEW_SCHEMA")))
+            (jdbc/execute! conn-spec "CREATE SCHEMA NEW_SCHEMA;")
+            (is (contains? (set (get-schemas)) "NEW_SCHEMA")))
+          (testing "drop schema"
+            (jdbc/execute! conn-spec "DROP SCHEMA NEW_SCHEMA;")
+            (is (not (contains? (get-schemas) "NEW_SCHEMA"))))
+          (testing "create table"
+            (is (not (contains? (get-tables) "NEW_TABLE")))
+            (jdbc/execute! conn-spec "CREATE TABLE NEW_TABLE (id INTEGER);")
+            (is (contains? (get-tables) "NEW_TABLE"))
+          (testing "drop table"
```

Comment:
tests for add column, drop column too?

Reply:
Done [Add test for adding/removing columns](https://github.com/metabase/metabase/pull/29826/commits/eed8cad480534e612cf5c1f8ac685f43e258af81)